ext/native-decls/GetVehicleNumberPlateText.md

```diff
@@ -8,8 +8,10 @@ apiset: server
 char* GET_VEHICLE_NUMBER_PLATE_TEXT(Vehicle vehicle);
 ```
 
-
 ## Parameters
 * **vehicle**: 
 
+## Note
+If the number plate is less than 8 character return value will have a space at the end to fill the remaining characters
```

Comment:
Probably better 🤔 :
`If the number plate text is less than 8 characters the result will be appended with spaces until there is 8 characters in it.`

Reply:
I also think that the notes are supposed to come before the ##Paramaters, and wont take a `##` markup